### ISSUE TITLE
Any way to check if from/to values where changes by user?

### ISSUE THREAD

**Issue by JamesJGoodwin** [IonDen/ion.rangeSlider#523]
Hello! Is there any way I could check if user was changed the `options.from` or `options.to` options? I know, I could create an event `onChange` and if some value was changed then set the flag, for example, `isFromOptChanged = 1` and then track it, but asking just to be sure if there any native functionality to not reinvent bicycle.

**Comment by IonDen** [comment]
Hello, no. This functionality is not needed from the box. So, please, do your solution.

**Comment by slince** [comment]
up

**Comment by slince** [comment]
@IonDen  Hi, can you give some suggestion for the question?

**Comment by IonDen** [comment]
Hi, this question is out of scope. This is a general programming question.
Next time ask it on StackOverflow :)

Solution is like this:
http://jsfiddle.net/1wox34nh/

**Comment by mejiafabiandj** [comment]
maybe you can check which slider contains the "type_last" class

![image](https://user-images.githubusercontent.com/3708192/39656338-b3dc0472-4fc4-11e8-9dca-c1e422d131b9.png)

**Comment by JamesJGoodwin** [comment]
Well, I can see that this issue is some kind of interesting to resolve for some people. Here's how I resolved it. 

Let's imagine that we have an array with different numerical values that I'm using as values in my instance of ionRangeSlider. I can find it's **minimum** and **maximum** values by very well known JavaScript function. Since I have those numbers I could check if current `from` option is different from my local **minimum** value and option `to` is different from my local **maximum** value. At any time. And if they're not similar, I can make a conclusion that my slider was changed somehow(by user or programatically).

I guess that this issue should be closed 'cause, as it was said earlier, this question is out of scope. I'll close the issue now, but feel free to ask any question you what.
